README.md

```diff
@@ -197,6 +197,45 @@ further reduces the maximum PR title length to ~65 characters. PRs violating
 these limits are labeled `M-failed-description` and are not merged.
 
 
+### Message attributes
+
+Commit description may contain special attributes, which are recognized and
+processed by the bot. If the bot cannot parse an attribute, it marks the PR
+with `M-failed-description` label.
+
+1.`*Authored-by*` - The commit message author.
+We need to provide this attribute when the actual PR author is different from
+the author automatically provided by GitHub (which is the author of the PR branch
+first commit). The attribute should be specified at the beginning of the PR
```

Comment:
Please double check the "author of the PR branch first commit" statement. IIRC, you have [concluded](https://measurementfactory.atlassian.net/browse/SQUID-642?focusedCommentId=46179) that it is the PR author that is used as the merge commit author. This README file also says that the staged commit author is "The PR author object".

Please do not change the text even if my suspicions are correct -- I am editing this text now. Just let me know which author is used by default, first branch commit or PR. Thank you.

Reply:
> author of the PR branch first commit

My experiment showed that the PR author is used by default (by us and by GitHub when creating merge commit). So my previous statement was wrong - "author of the PR branch first commit" has nothing to do with this. If we retrieve a PR (via API), the PR author is stored in the `user.login` field (and returned by our _prAuthor()).